src/base/bittorrent/torrentimpl.cpp

```diff
@@ -1188,7 +1195,16 @@ qlonglong TorrentImpl::eta() const
                 seedingTimeEta = 0;
         }
 
-        return std::min(ratioEta, seedingTimeEta);
+        qlonglong inactiveSeedingTimeEta = MAX_ETA;
+
+        if (maxInactiveSeedingTimeValue >= 0)
+        {
+            inactiveSeedingTimeEta = (maxInactiveSeedingTimeValue * 60) - timeSinceActivity();
+            if (inactiveSeedingTimeEta < 0)
+                inactiveSeedingTimeEta = 0;
```

Comment:
I like that suggestion but std::max got upset because it was trying to compare a qlonglong to an int.

Changed the 0 to be wrapped as a qlonglong. I have very little C++ experience so I'm not sure if there's a better way to make it work.

`inactiveSeedingTimeEta = std::max(inactiveSeedingTimeEta, qlonglong(0));`

Reply:
Yep! That did the trick